Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -18,6 +18,12 @@
 #import "CDTHTTPInterceptor.h"
 #import "CDTLogging.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, nonatomic, strong) NSURLResponse *response;
```

Comment:
it does, here: https://github.com/cloudant/CDTDatastore/pull/190/files#diff-e78ad7a4607eb8512ef453210d9c08f6R123